school-finder-frontend/app/index.html

```diff
@@ -2,6 +2,7 @@
 <html ng-app="sfApp">
 <head>
 	<title>School Finder</title>
+    <link href="/favicon.ico" type="image/x-icon" rel="icon" />
```

Comment:
This won't resolve correctly unless the app is run off of a root path as it's an absolute path. 
If we switch this to be `href="favicon.ico"` that should do the trick.

Also indentation.